### PR TITLE
send mail with sidekiq

### PR DESCRIPTION
**Pull request by nhunght**

### CODE REVIEW

**Comment by truonglocbinh** [comment]
ok

File: Gemfile [hunk 1]
```diff
@@ -13,11 +13,16 @@ gem "config"
 gem "kaminari", "~> 0.16.1"
 gem "cancancan", "~> 1.10"
 gem "ransack"
+gem "sidekiq"
+gem "redis-rails"
+gem "sinatra", ">= 1.3.0", :require => nil
```
Comment (dieunb) [inline Gemfile]: Convention

File: app/models/exam.rb [hunk 2]
```diff
@@ -23,6 +25,10 @@ class Exam < ActiveRecord::Base
     end
   end
 
+  def send_email_result_exam
```
Comment (dieunb) [inline app/models/exam.rb]: Private

File: config/initializers/smtp_settings.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+ActionMailer::Base.smtp_settings = {
+  address: "smtp.gmail.com",
+  port: 587,
+  domain: "gmail.org",
+  user_name: "linhnhungnguyen94@gmail.com",
```
Comment (dieunb) [inline config/initializers/smtp_settings.rb]: lưu environment variable nhé

File: config/environments/development.rb [hunk 4]
```diff
@@ -1,41 +1,33 @@
...
-  # Raises helpful error messages.
   config.assets.raise_runtime_errors = true
 
-  # Raises error for missing translations
-  # config.action_view.raise_on_missing_translations = true
+  ActionMailer::Base.smtp_settings = {
+    address: "smtp.gmail.com",
+    port: 587,
+    domain: "gmail.org",
+    user_name: "linhnhungnguyen94@gmail.com",
```
Comment (dieunb) [inline config/environments/development.rb]: Lưu biến environment nhé

File: config/environments/development.rb [hunk 5]
```diff
@@ -1,41 +1,33 @@
...
+    password: "nhung1609",
```
Comment (dieunb) [inline config/environments/development.rb]: Lưu biến environment nhé

**Pull request merged**